src/components/Shelf/Shelf.tsx

```diff
@@ -71,22 +71,23 @@ const Shelf = ({
 
   const renderTile = useCallback(
     (item: PlaylistItem, isInView: boolean) => {
+      const url = mediaURL({ media: item, playlistId: playlist.feedid, play: type === PersonalShelf.ContinueWatching });
       return (
         <Card
           key={item.mediaid}
           progress={watchHistory ? watchHistory[item.mediaid] : undefined}
-          onClick={isInView ? () => onCardClick(item, playlist.feedid, type) : undefined}
           onHover={typeof onCardHover === 'function' ? () => onCardHover(item) : undefined}
           featured={featured}
           disabled={!isInView}
           loading={loading}
           isLocked={isLocked(accessModel, isLoggedIn, hasSubscription, item)}
           posterAspect={posterAspect}
           item={item}
+          redirectUrl={isInView ? url : undefined}
```

Comment:
I don't think we still need this isInView condition. iirc it was to prevent accidental clicks of the cards on the edges under the arrows. Hopefully the links work better with the slider? Can you test and verify?

Reply:
Unfortunately, the behavior for the slider is the same
https://github.com/jwplayer/ott-web-app/assets/118803926/d8cec1c7-df6e-44fe-a707-3a2270af906c

Reply:
OK, then I will create a separate ticket for this w/ bot detection. Thank you for checking!